### PR TITLE
Update toolcombobox.py

### PR DESCRIPTION
**Pull request by Hrishi1999**

### CODE REVIEW

File: src/sugar3/graphics/toolcombobox.py [hunk 1]
```diff
@@ -18,15 +18,32 @@
...
 
 class ToolComboBox(Gtk.ToolItem):
+    '''
+    UI for ToolComboBox
+    A ToolComboBox is a widget that allows the user to choose
+    from a list of valid choices. The ToolComboBox displays
+    the selected choice. When activated, it displays a popup
+    which allows the user to make a new choice.
+    Functions:
+        ToolComboBox.new()
```
Comment (samdroid-apps) [inline src/sugar3/graphics/toolcombobox.py]: Just use the constructor

File: src/sugar3/graphics/toolcombobox.py [hunk 2]
```diff
@@ -18,15 +18,32 @@
...
+            Returns a new tool combobox.
+    Keyword Args:
```
Comment (samdroid-apps) [inline src/sugar3/graphics/toolcombobox.py]: Please document the label-text prop as well

**Comment by samdroid-apps** [comment]
Your documentation does not help me understand how to use the module.  You may need to provide an example, as well as documenting how the use the `.combo` attribute.

**Comment by Hrishi1999** [comment]
Can you help me explain combo attr. & label-text?
Thanks

**Comment by samdroid-apps** [comment]
@Hrishi1999 the combo attribute is like a GtkComboBox, so either what the api consumer passed or an auto made one.

Label text is probably the text of the label.  Please experiment to see what it does and where it is.

**Comment by Hrishi1999** [comment]
Anyone?

**Comment by Hrishi1999** [comment]
Today is the last day for GCI.

**Pull request closed without merging**